### PR TITLE
Alert: run timer in useEffect to prevent multiple triggers

### PR DESCRIPTION
**Pull request by goapunk**

### CODE REVIEW

File: adhocracy4/static/Alert.jsx [hunk 1]
```diff
@@ -1,11 +1,17 @@
-import React from 'react'
+import React, { useRef, useEffect } from 'react'
 import django from 'django'
 
 const Alert = ({ type, message, onClick, timer }) => {
+  const timer0 = useRef()
```
Comment (khamui) [inline adhocracy4/static/Alert.jsx:5]: maybe a different naming instead of timer0? 

for instance the argument `timeInMs` and the actual ref `timer`?

**Review by khamui — APPROVED** [review]
Good stuff @goapunk ! just a tiny little thing about the naming:

File: adhocracy4/static/Alert.jsx (hunk 1 above)
Comment (goapunk) [reply adhocracy4/static/Alert.jsx:5]: yeah, good idea! `timer` and `timer0` is not a good choice

Comment (khamui) [reply adhocracy4/static/Alert.jsx:5]: i know we would need to change it where we use it. or you "rename" it here for instance ({ type, message, onClick, timer: timeInMs }). I admit its sketchy.

Comment (goapunk) [reply adhocracy4/static/Alert.jsx:5]: we only use it in kosmo, right? I can change it there

Comment (goapunk) [reply adhocracy4/static/Alert.jsx]: opened https://github.com/liqd/a4-kosmo/pull/203 for it

Comment (khamui) [reply adhocracy4/static/Alert.jsx]: perfect thanks!

**Pull request merged**